src/pyflex/window_selector.py

```diff
@@ -911,12 +992,36 @@ def plot(self, filename=None):
         ax.spines['right'].set_color('none')
         ax.spines['left'].set_color('none')
         ax.spines['top'].set_color('none')
+        stepsize = 200
+        start, end = ax.get_xlim()
+        ax.xaxis.set_ticks(np.arange(start, end+1, stepsize))
```

Comment:
This is dangerous. It will work for continental and global scale waveforms but not local ones where the whole waveform is only a couple of seconds.

If you want to tune the ticks please use one of matplotlibs tick locators. Otherwise I think this can also be reverted as matplotlib's default ticks are usually quite ok.